fabtests/benchmarks/benchmark_shared.c

```diff
@@ -310,12 +322,17 @@ int bandwidth(void)
 				ret = ft_post_inject_buf(ep, remote_fi_addr,
 						opts.transfer_size, NO_CQ_DATA,
 						tx_ctx_arr[j].buf, tx_seq);
+			} else if (opts.use_fi_more) {
+				flags = set_fi_more_flag(i, j, flags);
+				ret = ft_sendmsg(ep, remote_fi_addr,
+						 tx_ctx_arr[j].buf,
+						 opts.transfer_size,
+						 &tx_ctx_arr[j].context, flags);
 			} else {
-				ret = ft_post_tx_buf(ep, remote_fi_addr,
-						opts.transfer_size, NO_CQ_DATA,
-						&tx_ctx_arr[j].context,
-						tx_ctx_arr[j].buf,
-						mr_desc, tx_seq);
+				ret = ft_post_tx_buf(
```

Comment:
Nit: it seems you don't have to touch this code block? Leaving them unchanged will make it easier to review.